<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="produto">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>1130</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 35pt &quot;Courier&quot;;
background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="empresa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;

background-color: rgb(223, 213, 255);

border-color: rgb(162, 162, 162);

border-style: Outset;
border-width: 2px;
border-radius: 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Users/Helio/.designer/backup/CADASTRO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="foto_produto">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>88</y>
      <width>280</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 249, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_fecha">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>610</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;F10&gt;FECHAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mcodigo">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>610</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>13</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>610</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Codigo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sub_total">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>420</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
font: 700 25pt &quot;Courier&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>SUB-TOTAL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="logohti">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>470</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="usuario">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>350</y>
      <width>130</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(197, 205, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="operador">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>480</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="numero_pedido">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
font: 700 14pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
font: 900 40pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>320</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
font: 900 40pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Preco R$</string>
    </property>
   </widget>
   <widget class="QLabel" name="cabecalho">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>420</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_sair">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>610</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>CANCELAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_buscar_produto">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>610</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>   Buscar</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>420</width>
      <height>468</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
font: 9pt &quot;Courier&quot;;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>320</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
font: 900 40pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Total R$</string>
    </property>
   </widget>
   <widget class="QLabel" name="preco">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>320</width>
      <height>70</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(255, 255, 255);
font: 900 40pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="quantidade">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>320</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(255, 255, 255);
font: 900 40pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="total_itens">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>320</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(255, 255, 255);
font: 900 40pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>mcodigo</tabstop>
  <tabstop>bt_buscar_produto</tabstop>
  <tabstop>bt_fecha</tabstop>
  <tabstop>bt_sair</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
